src/cloud-providers/gcp/manager.go

```diff
@@ -22,7 +22,7 @@ func (_ *Manager) ParseCmd(flags *flag.FlagSet) {
 	flags.StringVar(&gcpcfg.GcpCredentials, "gcp-credentials", "", "Google Application Credentials, defaults to `GCP_CREDENTIALS`")
 	flags.StringVar(&gcpcfg.ProjectId, "gcp-project-id", "", "GCP Project ID")
 	flags.StringVar(&gcpcfg.Zone, "gcp-zone", "", "Zone")
-	flags.StringVar(&gcpcfg.ImageName, "gcp-image-name", "", "Pod VM image name")
+	flags.StringVar(&gcpcfg.ImageName, "image-name", "", "Pod VM image name")
 	flags.StringVar(&gcpcfg.MachineType, "gcp-machine-type", "e2-medium", "Pod VM instance type")
```

Comment:
Also removing `gcp` prefix for `image-name` but keeping it for `gcp-machine-type` makes things even more random and inconsistent 🤔 …

… which you address in the next commit.